backend/src/main/java/pl/edu/agh/awiteks_backend/models/ForumThread.java

```diff
@@ -0,0 +1,55 @@
+package pl.edu.agh.awiteks_backend.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+@JsonIgnoreProperties({"user"})
+public class ForumThread extends AbstractModel<ForumThread>{
```

Comment:
Tu w sumie jak `ForumPost` nie trzeba rozszerzać `AbstractModel` tylko zrobić pola `id` i `title` jako nazwę tematu
```java
private Integer id;
private string title;
```
Ja i tak zamierzam wyrzucić `AbstractModel` bo nie spina mi się to tak fajnie z bazą, jak myślałem

Reply:
Done